hybrid/compartment_examples/inter_comp_call/secure/compartments.s

```diff
@@ -18,14 +18,14 @@ comp_f_fn:
     // `switch_compartment`.
     str       clr, [sp, #-16]!
 
-    // Retrieve local capability containing switcher information for `ldpblr`
+    // Retrieve local capability containing switcher information for `pdlblr`
```

Comment:
Was this always `pdlblr`? If so, it should be `ldpblr`.

Reply:
Oops, I thought I'd fixed it (on master this is only misspelled as `pdlblr` in `secure` and `secure-update_ddc`)

Reply:
Fixed in c4d34fed27a2dfba8bddb2dfbd9060dcce53d379